src/components/AudioInfo.vue

```diff
@@ -663,24 +663,50 @@ export default defineComponent({
       () => supportedFeatures.value?.synthesisMorphing
     );
 
-    const isValidMorphingInfo = computed(() => {
+    const isInvalidMorphingInfo = computed(() => {
       if (audioItem.value.morphingInfo == undefined) return false;
       return !store.getters.VALID_MOPHING_INFO(audioItem.value);
     });
 
     const mophingTargetEngines = store.getters.MORPHING_SUPPORTED_ENGINES;
 
     const mophingTargetCharacters = computed(() => {
-      const allCharacters = store.getters.GET_ORDERED_ALL_CHARACTER_INFOS;
-      return allCharacters
-        .map((character) => {
+      // 選択可能なスタイルをフィルタリングする.
+      const baseEngineId = audioItem.value.engineId;
+      const baseStyleId = audioItem.value.styleId;
+      if (baseEngineId === undefined || baseStyleId == undefined) {
+        return [];
+      }
```

Comment:
基本、想定外の挙動は全部throwが良いかなと思います！（ログに出るので。実装方針で案内できてなくて申し訳ない 
🙇  ）
```suggestion
      if (baseEngineId == undefined || baseStyleId == undefined) {
        throw new Error(
          "baseEngineId == undefined || baseStyleId == undefined"
        );
      }
```